/pilot_js/pilot_js_node:
  ros__parameters:
    js_steering_flipped: true
    js_throttle_flipped: false
    
    js_steering_axis: 0
    js_throttle_axis: 4

    js_upshift_button: 5
    js_downshift_button: 4

    max_wheel_angle_rad: 0.6 # Maximum angle (left and right) that the vehicle wheel can swing to
    wheel_angle_change_rate: 0.1 # Rate at which the wheel steers

/joy/js_node:
  ros__parameters:
    sticky_buttons: true # Holding down a button will only trigger certain event once
    coalesce_interval_ms: 10
